source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+When you document a new server parameter, include the following details 
+in addition to specific information about the new parameter. These details
+ought to be documented in the order they appear in the following table:
```

Comment:
[Minor suggestion]

Consider addressing the writer here:

"Document these details in the order they appear in the following table: "

Reply:
Done. Thanks!